src/accapi/client.py

```diff
@@ -307,14 +311,7 @@ def start(
         self._socket.settimeout(1)
         self._connectionId = None
         self._writable = False
-        self._request_connection(
```

Comment:
I have not found the rational behind requiring that the _request_connection be executed inside the client thread. This was omitted from #4, please comment on that PR if there's a good reason to move it.